### DISCUSSION TITLE
How to import CSV data into Photon

### DISCUSSION THREAD

**Discussion by 861** [komoot/photon#807] [Q&A]
Hi, 

I have some address data stored in a CSV file, I already finished the setup of Photon, but want to get a more accurate result. 
Any advice to import the CSV data?

**Comment by felix-geovelo** [comment]
I've done it this way:
- I've imported extra data from a CSV in Nominatim (following what's done for TIGER data and adapt it for my needs). 
- I then create necessary code in Photon to import these new data comming from Nominatim -for import and update process).

You'll have to get your hand dirty for this i think (and also recreate the index)
